test/jdk/javax/swing/border/Test4760089.java

```diff
@@ -21,29 +21,51 @@
  * questions.
  */
 
+import java.awt.Dimension;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JComponent;
+import javax.swing.JPanel;
+import javax.swing.border.Border;
+import javax.swing.border.EtchedBorder;
+import javax.swing.border.TitledBorder;
+
 /*
  * @test
  * @bug 4760089
  * @summary Tests that titled border do not paint inner titled border over its caption
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test4760089.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual Test4760089
  */
 
-import javax.swing.JApplet;
-import javax.swing.JPanel;
-import javax.swing.border.Border;
-import javax.swing.border.EtchedBorder;
-import javax.swing.border.TitledBorder;
+public class Test4760089 {
+    public static void main(String[] args) throws Exception {
+        String testInstructions = """
```

Comment:
You have lost the first part of the instructions.
It used to start with
"When applet starts, you'll see a panel with a compound titled border."

now this would be 
"When the test starts, ..."

Probably you'll need to add an extra row to the instructions.

Reply:
Updated.